addons/explosives/stringtable.xml

```diff
@@ -1038,5 +1038,11 @@
             <Polish>Czasomierz Wybuchu</Polish>
             <Russian>Таймер взрывчатки</Russian>
         </Key>
+        <Key ID="STR_ACE_Explosives_IsEOD_DisplayName">
+            <English>Is EOD</English>
```

Comment:
```suggestion
            <English>Is EOD</English>
            <German>Kampfmittelbeseitigung</German>

Reply:
Sounds like a civilian job description. Usually in the BW they are called the same as in English EOD